Util/MSBuildCommunityTasks/MSBuild.Community.Tasks.Targets

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Is this something which we can pull from NuGet instead of including it in the source? Also, is it x-plat or are we expecting to only do staging from Windows machines?

Reply:
Yes, it is x-plat. I can look at pulling it from nuget instead, I forgot about nuget when importing it.